hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -444,4 +446,50 @@ public String getCanonicalServiceName() {
     return proxy.getCanonicalServiceName();
   }
 
+  /**
+   * Add acl for Ozone object. Return true if acl is added successfully else
+   * false.
+   * @param obj Ozone object for which acl should be added.
+   * @param acl ozone acl top be added.
+   *
```

Comment:
Can you add javadoc for the @return values? Same for the other ACL APIs.

Reply:
done